cli/src/create.rs

```diff
@@ -0,0 +1,189 @@
+use std::{env, error::Error, fs, path::Path};
+
+use dialoguer::{theme::ColorfulTheme, Select};
+use include_dir::{include_dir, Dir};
+
+use crate::print::{print_create_android_success_message, print_create_ios_success_message};
+use crate::style::{self};
+
+// TODO: add  "react-native", "flutter"
+const TEMPLATES: [&str; 2] = ["ios", "android"];
+
+pub fn create_project(arg_platform: &Option<String>) -> Result<(), Box<dyn Error>> {
```

Comment:
You should consider using the [`anyhow`](https://docs.rs/anyhow/latest/anyhow/) crate for errors. Here you're returning a memory reference to the dynamic `Error` trait. The `anyhow` crate wraps that complexity in a trait compatible with all `std` api's.

```suggestion
pub fn create_project(arg_platform: &Option<String>) -> anyhow::Result<()> {
```